pyproject.toml

```diff
@@ -94,13 +140,21 @@ use_parentheses = true
 ensure_newline_before_comments = true
 line_length = 88
 
-[build-system]
-requires = ["poetry>=0.12"]
-build-backend = "poetry.masonry.api"
 
 [tool.mypy]
+exclude = ".venv/.*"
+implicit_reexport = true
 mypy_path = "./typings"
-strict = true
```

Comment:
Any reason to remove "strict" and "show_column_numbers"?

Reply:
I think they just got reordered.  Not sure why that was